decosjoin/tests/test_api.py

```diff
@@ -57,6 +64,7 @@ def test_status(self):
 
     @patch("decosjoin.server.DecosJoinConnection._get_response", get_response_mock)
     @patch("decosjoin.server.DecosJoinConnection._post_response", post_response_mock)
+    @patch('decosjoin.api.decosjoin.decosjoin_connection.page_size', 10)
     def test_getvergunningen(self):
         response = self._client_get("/decosjoin/getvergunningen")
         self.assertEqual(response.status_code, 200, response.data)
```

Comment:
Passing response.data is for debugging I presume, correct?

Reply:
correct